slices/libpam-runtime.yaml

```diff
@@ -0,0 +1,67 @@
+package: libpam-runtime
+
+essential:
+  - libpam-runtime_copyright
+
+slices:
+  bins:
+    essential:
+      - libpam-modules_libs
+    contents:
+      # perl scripts that rely on debconf to properly work and discover
+      # plugins/rules in /usr/share/pam-configs/*
+      /usr/sbin/pam-auth-update:
+      /usr/sbin/pam_getenv:
+
+  config:
+    contents:
+      /etc/pam.conf:
+      /etc/pam.d/other:
+
+  # folders expected by libpam to exist
+  var:
+    contents:
+      /var/lib/pam/:
+
+  # Used by pam-auth-update to generate the pam.d/ config
+  # files inside etc. However pam-auth-update relies on
+  # debconf to actually work and discover these packages.
```

Comment:
That wouldn't work, since we don't install packages through dpkg, pam-auth-update won't be able to iterate packages and find which ones have these templates. This must be emulated instead.

Reply:
Does it make sense to keep this slice then? If the delivered binary is not gonna be functional...

Reply:
That is a good point, I guess it makes sense to remove it completely. I'll leave a comment in its place